src/timelines/honeycombBlock.js

```diff
@@ -4,8 +4,16 @@ import { config, eventCodes, taskSettings } from "../config/main";
 import { photodiodeGhostBox, pdSpotEncode } from "../lib/markup/photodiode";
 import { buildFixationTrial } from "../trials/fixation";
 
+/**
+ * Builds the blocks of trials that form the core of the Honeycomb experiment
+ *
+ * 1) A fixation dot is shown at the center of the screen
+ * 2) The stimulus image is shown and the user is prompted to press the correct key
+ *
+ * Note that the block is conditionally rendered and repeated based on the task settings
+ */
 function buildHoneycombBlock(jsPsych) {
-  const { honeycomb: honeycombSettings } = taskSettings;
+  const honeycombSettings = taskSettings.honeycomb;
```

Comment:
Nitpicky but it might be easier to read if config variables are in caps?

Reply:
Definitely on my nice to haves but I'm not sure where to draw the line. Maybe just `SETTINGS`, `LANGUAGE`, and `ENV`? At the very least I think `env` isa  much better name than `config`